### ISSUE TITLE
variable which contains all valid variables?

### ISSUE THREAD

**Issue by jorhett** [tomasbjerre/pull-request-notifier-for-bitbucket#114]
Can we get a variable for the URL which contains all of the possible variables in url-escaped key=value format?  Would be nice to just set this:

```
http://jenkins-master1.services.ooyala.net:8080/git/notifyCommit?url=${PULL_REQUEST_TO_SSH_CLONE_URL}&sha1=${PULL_REQUEST_FROM_HASH}&${KITCHEN_SINK_OF_VALUES}
```

**Comment by tomasbjerre** [comment]
Yes, good idea!

**Comment by tomasbjerre** [comment]
Released in 2.18
